patches/ordered-binary+1.2.5.patch

```diff
@@ -0,0 +1,181 @@
+diff --git a/node_modules/ordered-binary/index.js b/node_modules/ordered-binary/index.js
```

Comment:
This package is used for the LMDB key encoding, but is incompatible with the SES shim. I'm still unsure why I needed to patch it as it should have loaded pre-lockdown. We could probably try to figure out a way to avoid it entirely since we only use string keys, but patching was the path of least resistance for me.

Reply:
It was another case of `lmdb` not being loaded early enough.